packages/esbuild-plugin/examples/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Since we are not installing deps in example apps, we need this eslint configuration to ignore the _dependency not found_ error.